.bazelrc

```diff
@@ -1 +1 @@
-test --features=race --test_output=errors
+test --@io_bazel_rules_go//go/config:race --test_output=errors
```

Comment:
NOTE: this change is optional but it's introduced in [rules_go v0.23.0](https://github.com/bazelbuild/rules_go/releases/tag/v0.23.0) and likely will be the future.